test/test_posts.py

```diff
@@ -0,0 +1,128 @@
+import pytest
+from app import model, schemas
+
+
+def test_get_all_post(authorized_client,test_posts):
+    res = authorized_client.get("/posts/all")
+    def validate(post):
```

Comment:
why function within function, is there a specific reason?

Reply:
I try to map posts but for some reason that does not working and i forgot to comment this function